start_esp/start_esp.py

```diff
@@ -533,6 +534,11 @@ def make_argparser():
         default=DEFAULT_PID_FILE,
         help=argparse.SUPPRESS)
 
+    # always_print_primitive_fields.
+    parser.add_argument('--transcoding_always_print_primitive_fields',
```

Comment:
could you use 

action='store_true'

to be consistent with Boolean flags.

It means:  if the flag is not set,  its value is false,  
but if you just do --transcoding_always_print_primitive_fields,  it will set the flag to true. you can also do
--transcoding_always_print_primitive_fields=true or false

Reply:
@qiwzhang Yep, can do...